### PR TITLE
docs: fix br tools compatibility

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-overview.md [hunk 1]
```diff
@@ -122,14 +122,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
+> 
+在执行备份和恢复操作之前，BR（Backup & Restore）工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容的问题。
 
-在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+从 TiDB v6.7.0 版本开始，我们逐步支持通过 SQL 语句来执行备份和恢复操作。因此，我们强烈建议您在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
+
+
+TiDB v6.6.0 版本之前的 BR 版本兼容性矩阵将保持不变：
```
Comment (BornChanger) [inline br/backup-and-restore-overview.md:132]:
```suggestion
TiDB v6.6.0 版本之前的 BR 版本兼容性矩阵：
```

File: br/backup-and-restore-overview.md [hunk 2]
```diff
@@ -122,14 +122,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
 
 > **注意：**
->
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
+> 
+在执行备份和恢复操作之前，BR（Backup & Restore）工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容的问题。
 
-在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+从 TiDB v6.7.0 版本开始，我们逐步支持通过 SQL 语句来执行备份和恢复操作。因此，我们强烈建议您在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```
Comment (BornChanger) [inline br/backup-and-restore-overview.md:129]: 这句话放矩阵后面.

**Review by BornChanger — APPROVED** [review]

File: br/backup-and-restore-overview.md [hunk 3]
```diff
@@ -122,14 +122,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 ### 版本间兼容性
 
 > **注意：**
->
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
+> 
+在执行备份和恢复操作之前，BR（Backup & Restore）工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容的问题。
```
Comment (lilin90) [inline br/backup-and-restore-overview.md:127]:
```suggestion
>
> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。

在执行备份和恢复操作之前，BR (Backup & Restore) 工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容。
```

File: br/backup-and-restore-overview.md (hunk 2 above)
Comment (lilin90) [inline br/backup-and-restore-overview.md:129]: 这里是想说 v7.0？
```suggestion
从 v7.0 版本开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```

**Comment by lilin90** [comment]
说明：因 [v7.0 文档已于 2023-08-24 归档](https://docs-archive.pingcap.com/)，不再接受任何更新，所以不再 cherry pick 到此版本。

File: br/backup-and-restore-overview.md (hunk 2 above)
Comment (benmaoer) [reply br/backup-and-restore-overview.md:129]: 不是 v7.0

Comment (benmaoer) [reply br/backup-and-restore-overview.md:129]: 建议放到矩阵前面，用户先看到新版本的情况，只有少量用户需要去看历史情况

File: br/backup-and-restore-overview.md [hunk 4]
```diff
@@ -123,13 +123,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 
 > **注意：**
 >
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
 
-在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+在执行备份和恢复操作之前，BR (Backup & Restore) 工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容。
+
+从 TiDB v6.7.0 版本开始，我们逐步支持通过 SQL 语句来执行备份和恢复操作。因此，我们强烈建议您在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```
Comment (benmaoer) [inline br/backup-and-restore-overview.md:130]:
```suggestion
从 TiDB v6.7.0 版本开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```

File: br/backup-and-restore-overview.md [hunk 5]
```diff
@@ -123,13 +123,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 
 > **注意：**
 >
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
 
-在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+在执行备份和恢复操作之前，BR (Backup & Restore) 工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容。
+
+从 TiDB v6.7.0 版本开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```
Comment (benmaoer) [inline br/backup-and-restore-overview.md:130]:
```suggestion
从 TiDB v7.0.0 版本开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
```

File: br/backup-and-restore-overview.md [hunk 6]
```diff
@@ -123,13 +123,18 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
 >
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
 
-在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+在执行备份和恢复操作之前，BR (Backup & Restore) 工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具将会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容。
+
+从 TiDB v7.0.0 版本开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这将有助于确保恢复操作的顺利执行和数据的一致性。
+
+
```
Comment (lilin90) [inline br/backup-and-restore-overview.md:132]: - 因不是本文首次提到，不需要再解释全称。
- 删除了多余空行。
```suggestion
在执行备份和恢复操作之前，BR 工具会检查自己的版本与 TiDB 集群的版本是否兼容。如果它们之间不兼容，BR 工具会报错并停止执行。如果你想跳过这个版本检查，可以设置 `--check-requirements=false`。但是需要注意，跳过检查可能会导致恢复的数据不兼容。

从 v7.0.0 开始，TiDB 逐步支持通过 SQL 语句来执行备份和恢复操作。因此，强烈建议在备份和恢复集群时使用与 TiDB 集群相同大版本的 BR 工具，并避免跨大版本进行数据备份和恢复操作。这有助于确保恢复操作的顺利执行和数据的一致性。

```

**Review by lilin90 — COMMENTED** [review]
/lgtm

File: br/backup-and-restore-overview.md [hunk 7]
```diff
@@ -123,13 +123,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 
 > **注意：**
 >
-> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+> 建议使用与 TiDB 集群相同大版本的 BR 工具进行集群的备份和恢复。
```
Comment (ran-huang) [inline br/backup-and-restore-overview.md:126]: “相同大版本”指的是 major.minor.patch 的 major 一致吗？

Comment (lilin90) [reply br/backup-and-restore-overview.md:126]: 我理解是的，@benmaoer 确认下呢？

Comment (benmaoer) [reply br/backup-and-restore-overview.md:126]: 是哒，比如 6.5.0 的备份支持恢复到 v6.5.1 或者 v6.5.2

**Review by ran-huang — APPROVED** [review]

**Pull request merged**